.env.default

```
@@ -1,7 +1,9 @@
-LOG_LEVEL='INFO'
-LOCAL_PATH=/
+LOG_LEVEL=INFO
 
 RABBITMQ_HOST=rabbitmq
 RABBITMQ_USER=alkemio-admin
 RABBITMQ_PASSWORD=alkemio!
-RABBITMQ_QUEUE="virtual-contributor-engine-openai-assistant"
+RABBITMQ_QUEUE=virtual-contributor-engine-openai-assistant
+RABBITMQ_RESULT_QUEUE=virtual-contributor-invoke-engine-result
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The RABBITMQ_RESULT_QUEUE key should go before the RABBITMQ_USER key

(UnorderedKey)